src/server/controllers/users.controller.js

```diff
@@ -34,3 +12,15 @@ export function getAll(req, res, next) {
     }
   });
 }
+
+// Get all user list except logined user
+export function getMentorList(req, res, next) {
+  user.find({ 'email' : { $ne : 'session@yoda.com' }},  // if session_auth add, 'session' => req.session.email
+    { email: 0, userId: 0, age: 0, region: 0, skills: 0, regDate: 0 }, (err, doc) => {
```

Comment:
This is super important. This subset of fields from user object is the set that gets exposed to Node (and thus all subsequent flows). Write that in the comment above this line.